qubesmanager/qube_manager.py

```diff
@@ -432,21 +513,14 @@ def check_updates(self):
                     pass
 
     def on_domain_added(self, _submitter, _event, vm, **_kwargs):
+        domain = self.qubes_app.domains[vm]
```

Comment:
Have you tested this with a dispvm that fails to start? I think this can cause a KeyError or QubesException if the VM is deleted immediately after failing to start.

Reply:
Yes, fixed.

Reply:
I noticed my fix causes addition of empty rows if the code raises an exception after creating it. I have to improve it.